tools/text_denormalization/pynini_export.py

```diff
@@ -44,7 +56,7 @@ def export_grammars(output_dir):
         for stage, fst in graphs.items():
             out_dir = os.path.join(output_dir, stage)
             os.makedirs(out_dir, exist_ok=True)
-            _generator_main(f"{out_dir}/{category}_{stage}.far", fst, category.upper())
+            _generator_main(f"{out_dir}/{category}_tmp.far", fst, category.upper())
 
```

Comment:
The dict `d` was needed when we have multiple fst for each stage, how about simplifying it to:

`    d = {}
    d['tokenize_and_classify'] = ClassifyFst().fst
    d['verbalize'] = VerbalizeFst().fst

    for category, fst in d.items():
        os.makedirs(output_dir, exist_ok=True)
        _generator_main(f"{output_dir}/{category}.far", fst, category.upper())`

Reply:
actually after looking at it again i prefer it this way with classify and verbalize folder, because these folders are mounted to the docker container where sparrowhawk expects the .far files to be, this is just mirroring the folderstructure of sparrowhawk